### ISSUE TITLE
Oops, something went wrong.  Please report this bug with the details below.

### ISSUE THREAD

**Issue by Naapi** [PowerShell/PSReadLine#1481]
-----------------------------------------------------------------------
Last 200 Keys:
```
 l e Enter
 I Backspace U n i n s t a l l - ; M o Backspace Backspace Backspace M O d Backspace Backspace o d Tab Tab Space A z u r e R M . - Backspace p r o i Backspace f i Backspace Backspace f i l e Enter
 UpArrow UpArrow UpArrow DownArrow UpArrow UpArrow UpArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow Space Backspace P Backspace Backspace P Enter
 UpArrow UpArrow UpArrow Enter
 Ctrl+v Enter
 Ctrl+v UpArrow DownArrow UpArrow LeftArrow Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace A z u r e R M . p r o f i l e Enter
 UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow Enter
 UpArrow UpArrow UpArrow UpArrow Space - F o p r Backspace Backspace r c e Enter
 Ctrl+v LeftArrow LeftArrow RightArrow Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Ctrl+v Enter
 U n Backspace Backspace Backspace UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow UpArrow Backspace
```

Exception:
```
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: left
Actual value was -2.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at System.Console.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.BackwardDeleteChar(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
-----------------------------------------------------------------------

**Comment by iSazonov** [comment]
See #1306

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.1 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions.
